.github/workflows/transformer-images.yaml

```
@@ -151,7 +145,8 @@ jobs:
             *.cache-to=type=gha,scope=transformer-images,mode=max
 
   release:
-    permissions: write # for release-action
+    runs-on: ubuntu-latest
+    permissions: write-all # for release-action
```

Too permissive, should be:

```suggestion
    permissions:
      # required to create GitHub release
      contents: write
```